src/back/services/notification.ts

```diff
@@ -29,22 +29,38 @@ import PushAPI = require('@pushprotocol/restapi')
 const chainId = isProdEnv() ? ChainId.ETHEREUM_MAINNET : ChainId.ETHEREUM_SEPOLIA
 const PUSH_CHANNEL_OWNER_PK = process.env.PUSH_CHANNEL_OWNER_PK
 const PUSH_API_URL = process.env.PUSH_API_URL
-const pkAddress = `0x${PUSH_CHANNEL_OWNER_PK}`
-const signer = NOTIFICATIONS_SERVICE_ENABLED ? new ethers.Wallet(pkAddress) : undefined
+
+function getSigner() {
+  console.log('notificationsServiceEnabled', NOTIFICATIONS_SERVICE_ENABLED)
+  if (NOTIFICATIONS_SERVICE_ENABLED) {
```

Comment:
I'd return 
```
if (!NOTIFICATIONS_SERVICE_ENABLED) {
  return undefined
}
```
first. So there is no if inside if. And you are also checking for the "negative" value first in the next condition.